aldryn_faq/models.py

```diff
@@ -189,15 +190,19 @@ class Meta:
 class QuestionListPlugin(CMSPlugin):
     questions = SortedManyToManyField(Question)
 
-    def __str__(self):
-        return str(self.questions.count())
-
     def copy_relations(self, oldinstance):
         self.questions = oldinstance.questions.all()
 
     def get_questions(self):
         return self.questions.all()
 
+    def __str__(self):
+        question_count = self.questions.count()
+        return '{0} {1} selected'.format(
+            question_count,
+            "question" if question_count == 1 else "questions"
+        )
+
```

Comment:
maybe.. but i don't I think is better to reuse but not required.

I wouldn't make it translatable as this is used for the object's internal representation, not very user facing at least not all the time.